schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species VARCHAR(255)
```

Comment:
- [ ] Instead of just adding the species column like this, it would be nice to add the actual command used to achieve this e.g.

```
   ALTER TABLE animals ADD species varchar(255);
```